JSDemos/Demos/PivotGrid/SimpleArray/description.md

```diff
@@ -1 +1,8 @@
-The PivotGrid is&nbsp;a&nbsp;high-performance, enterprise-ready UI component optimized for multi-dimensional data analysis. It&nbsp;allows your end users to&nbsp;easily arrange and summarize information in&nbsp;a&nbsp;cross-tabular format. In&nbsp;this demo, the PivotGrid&rsquo;s simplicity of&nbsp;use is&nbsp;illustrated. Sorting and filtering are enabled so&nbsp;you can sort data in&nbsp;ascending or&nbsp;descending order and by&nbsp;summary values. You can drag and drop fields to&nbsp;reorganize the PivotGrid and change filtering properties to&nbsp;analyze data more efficiently.
\ No newline at end of file
+The PivotGrid component can display data from an array of objects. Use the [dataSource](/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#dataSource) property to bind the PivotGrid to data. This property accepts a [PivotGridDataSource](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/) instance or its configuration object. Assign your array to this object's [store](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/store/) property.
+
+To display data in the PivotGrid, assign an array to the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) property. Each object in this array configures a single pivot grid field. Set the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.
```

Comment:
> To display data in the PivotGrid, **assign an array** to the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) property.

Which array I should assign? Is it the same array mentioned in the previous paragraph? Does this array have certain requirements or limitations?

> Each object in this array **configures a single pivot grid field**.

OK. But how exactly? I still don't see a link to the array element type.

> Set the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.

What should I assign to the `dataField` property?

Reply:
A summary from our personal discussion:

Write this:
> Assign a field name to the `DataField` property to populate the pivot grid field with data.

Instead of
> Set the `dataField` property  to populate the pivot grid field with data.